### DISCUSSION TITLE
画布中的节点在拖动时怎么设置才能让节点不覆盖其他节点？

### DISCUSSION THREAD

**Discussion by ElsaOOo** [antvis/X6#1912] [Q&A]
画布中的节点在拖动时怎么设置才能让节点不覆盖其他节点？

**Comment by NewByVector** [comment]
移动的节点处于画布上其他节点下方？

**Reply by ElsaOOo** [reply]
不是，移动的节点会出现在其他节点上方，需求是移动的时候不能两个节点重叠，
<img width="228" alt="image" src="https://user-images.githubusercontent.com/48074435/159385714-e1a09c2b-ddd1-49a1-8f36-6d14fa10c707.png">

像图片这样移动节点的时候，两个节点会重叠。

**Reply by NewByVector** [reply]
这个功能 x6 是不支持的，也没有比较好的方法来实现。😶

**Reply by GuangZ96** [reply]
应该只能通过调整"7-整理区"的位置来避免重叠
